applications/debug/unit_tests/subghz/subghz_test.c

```diff
@@ -654,6 +654,20 @@ MU_TEST(subghz_decoder_kinggates_stylo4k_test) {
         "Test decoder " SUBGHZ_PROTOCOL_KINGGATES_STYLO_4K_NAME " error\r\n");
 }
 
+MU_TEST(subghz_decoder_mastercode_test) {
+    mu_assert(
+        subghz_decoder_test(
+            EXT_PATH("unit_tests/subghz/mastercode_raw.sub"), SUBGHZ_PROTOCOL_MASTERCODE_NAME),
+        "Test decoder " SUBGHZ_PROTOCOL_MASTERCODE_NAME " error\r\n");
+}
+
+MU_TEST(subghz_decoder_mastercode_test2) {
```

Comment:
a test with 1 file, but working, is enough

Reply:
Ok, deleted the test2.

Reply:
Ok, I removed the test2